packages/module/src/ToolResponse/ToolResponse.tsx

```diff
@@ -80,11 +80,13 @@ export const ToolResponse: FunctionComponent<ToolResponseProps> = ({
               </div>
             )}
             {body && <div className="pf-chatbot__tool-response-body">{body}</div>}
-            <Card isCompact className="pf-chatbot__tool-response-card" {...toolResponseCardProps}>
-              <CardTitle {...toolResponseCardTitleProps}>{cardTitle}</CardTitle>
-              <Divider {...toolResponseCardDividerProps} />
-              <CardBody {...toolResponseCardBodyProps}>{cardBody}</CardBody>
-            </Card>
+            {cardTitle && cardBody && (
```

Comment:
Would we want to extend the ask a bit and allow rendering only title or body, rather than all-or-nothing?